project/weblap.html

```diff
@@ -1,136 +1,258 @@
 <!DOCTYPE html>
 <html lang="hu">
 	<head>
-	<meta charset ="utf-8">
-		<title>Rövid történetek</title>
-		<style>
-			body { 
-                background-color: black; 
-                color: white; 
+	<meta charset="utf-8">
+	<title>Rövid történetek</title>
+    <style>
+        body { 
```

Comment:
Bár nem ellenőriztem sorról sorra, de úgy tűnik, hogy a CSS fájlban ugyanez a tartalom. Ilyen esetben töltsd be azt. Illetve hagyhatod itt is a CSS-t (később fájni fog, de tapasztalatnak jó), viszont vagy az internal vagy az external css maradjon csak.

Reply:
igen, külön volt de azt is at akartam futtani a netes checken es faieletlt vele uh azert raktam vissza... :)